soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -2252,6 +2287,17 @@ RandomizerCheckObject Randomizer::GetCheckObjectFromActor(s16 actorId, s16 scene
                     break;
             }
             break;
+        case SCENE_KINSUTA:
```

Comment:
You shouldn’t need to have special cases here I don’t think, these should just go in the lookup table

Reply:
The problem with that is that `En_Sth` (the un-cursed skull people) have the appropriate RC check, but `En_Ssh` (the cursed ones) just display a single textbox regardless of params in vanilla. This special case is used to grab the check that you would get from the un-cursed forms to provide the hint as the cursed form.